packages/wrangler/src/__tests__/vitest.global.ts

```diff
@@ -4,4 +4,5 @@ export function setup(): void {
 	// This needs to be in a globalSetup script - it won't work in a setupFile script.
 	// https://github.com/vitest-dev/vitest/issues/1575#issuecomment-1439286286
 	process.env.LC_ALL = "C";
+	process.env.TZ = "UTC";
```

Comment:
Sometimes I see failures locally when dates are involved. This particular time it was the workflows tests. I think it's easier if we just set the time zone here.